paasta_tools/paasta_metastatus.py

```diff
@@ -601,7 +619,7 @@ def main():
         mesos_state=mesos_state,
     )
     metrics = get_mesos_stats()
-    mesos_metrics_status = get_mesos_metrics_health(mesos_metrics=metrics)
+    mesos_metrics_status = get_mesos_metrics_health(mesos_metrics=metrics, mesos_state=mesos_state)
```

Comment:
Personally, I think https://issues.apache.org/jira/browse/MESOS-6175 is going to be our best long-term solution. I'm also not sure how well hitting every slave will scale unless we do it in parallel. I also have a feeling we will get hit by timeouts and connectivity issues a bit more frequently, which might slow down this tool. My vote would be to save move this discussion to a separate issue/PR.